dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBMongoDB/AzureCosmosDBMongoDBMemoryRecord.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Linq;
+using Microsoft.SemanticKernel.Memory;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Microsoft.SemanticKernel.Connectors.AzureCosmosDBMongoDB;
+
+/// <summary>
+/// A MongoDB memory record .
+/// </summary>
+public sealed class AzureCosmosDBMongoDBMemoryRecord
```

Comment:
Does this need to be public? I believe the Memory store just exposes the MemoryRecord publicly, so if we can keep this internal, it reduces the possibility of breaking changes.

Reply:
Made it internal